core/src/main/scala/org/apache/spark/status/protobuf/RDDOperationGraphWrapperSerializer.scala

```diff
@@ -81,8 +80,8 @@ class RDDOperationGraphWrapperSerializer extends ProtobufSerDe {
   }
 
   private def serializeRDDOperationNode(node: RDDOperationNode): StoreTypes.RDDOperationNode = {
-    val outputDeterministicLevel = StoreTypes.RDDOperationNode.DeterministicLevel
-      .valueOf(node.outputDeterministicLevel.toString)
+    val outputDeterministicLevel = DeterministicLevelSerializer.serialize(
```

Comment:
Since `DeterministicLevelSerializer` is only used here, shall we move it into this file?